eleventy.config.ts

```diff
@@ -85,6 +85,23 @@ export default function (eleventyConfig: UserConfig) {
     filter: (path: string) => path.includes('src') || path.includes('images'),
   });
 
+  eleventyConfig.addFilter("shuffle", (arr: Array<any>) => {
```

Comment:
If you do keep this filter, move it to the `/src/_11ty/filters.ts` file to keep it with the others.